src/Microsoft.AspNetCore.Mvc/project.json

```diff
@@ -16,6 +16,7 @@
     "Microsoft.AspNetCore.Mvc.Formatters.Json": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Localization": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Razor": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*",
```

Comment:
+1 we have left the MVC tag helpers as optional to date. There's only two included by default and they aren't in the TagHelpers package.